package.json

```diff
@@ -69,7 +69,7 @@
     "tsconfig-paths-webpack-plugin": "^3.5.1",
     "tslib": "^2.2.0",
     "typescript": "^4.3.2",
-    "typings-for-css-modules-loader": "^0.2.0",
+    "typings-for-css-modules-loader": "^1.0.0",
```

Comment:
> json5  <1.0.2
Severity: high
Prototype Pollution in JSON5 via Parse Method - https://github.com/advisories/GHSA-9c47-m6qq-7p4h
fix available via `npm audit fix --force`
Will install typings-for-css-modules-loader@0.1.1, which is a breaking change
node_modules/typings-for-css-modules-loader/node_modules/json5
  loader-utils  <=1.4.0
  Depends on vulnerable versions of json5
  node_modules/typings-for-css-modules-loader/node_modules/loader-utils
    typings-for-css-modules-loader  >=0.2.0
    Depends on vulnerable versions of loader-utils
    node_modules/typings-for-css-modules-loader